core/models/user.py

```diff
@@ -1,5 +1,12 @@
 from django.contrib.auth.models import AbstractUser
+from django.db import models
 
 
 class User(AbstractUser):
-    pass
+    country = models.ForeignKey(
+        "Country", null=True, blank=True, on_delete=models.CASCADE
+    )
+    is_stakeholder = models.BooleanField(default=False)
```

Comment:
Hmmm I don't know if this is the best approach for user roles. Adding a new column per user role is not optimal because we need 4 columns to store one value (the role), this is a bad practice and it might create overhead in the future updates (adding / removing / updating a role / permission)
I think a better solution is to use Django's groups and permissions or add a _"type"_ field in User table
@sorinstelian What do you think?